.babelrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
@jmcolella yeah I'm not sure to be honest - when I remove the `.babelrc` the js doesn't compile when the tests run but I don't know why. Although I did discover that you can put the presets in `package.json` instead of creating a .babelrc so I've done that instead